documentation/release_notes.rst

```diff
@@ -8,6 +8,42 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6
+=============
+
+New Features
+------------
+- Added one more thread parallel backend. It is based on OpenMP 5.0. To enable the OpenMP backend, please define macro ``ONEDPL_USE_OPENMP_BACKEND`` into 1.
+- Added the range-based version of the ``reduce_by_segment`` algorithm and improved performance of the iterator-based ``reduce_by_segment`` APIs. 
+  Please note that the use of the ``reduce_by_segment`` algorithms requires C++17.
+- Added the serial-based versions of the following algorithms: ``for_each_n``, ``copy``,
+  ``copy_backward``, ``copy_if``, ``copy_n``, ``is_permutation``, ``fill``, ``fill_n``, ``move``, ``move_backward``.
+  For the detailed list, please refer to `Tested Standard C++ API Reference`_. 
+
+Changes affecting backward compatibility
+----------------------------------------
+- Fixed ``param_type`` API of random number distributions to satisfy C++ standard requirements.
+  The new definitions of param_type are not compatible with incorrect definitions in previous library versions.
+  Recompilation is recommended for all codes that might use param_type
+
+Fixed Issues
+------------
+- Fixed hangs and errors when oneDPL is used together with oneMKL in DPC++ programs.
+- Fixed possible data races in the following algorithms used with DPC++ execution policies: ``sort``, ``stable_sort``, ``partial_sort``, ``nth_element``.
+
+Known Issues and Limitations
+----------------------------
+- No new issues in this release.
+
+Existing Issues
+^^^^^^^^^^^^^^^
+Description of several issues was moved to `Known Limitations`_ section of oneDPL Library Guide. 
```

Comment:
Added link and made the naming consistent

Reply:
I am a bit confused with "oneDPL Library Guide", because DPL acronym already has "Library" in it. Would it be better to write "oneAPI DPC++ Library (oneDPL) Guide" or even "DPC++ Library (oneDPL) Guide"?

Reply:
Maybe we should consult with the infodev team what's most appropriate.

Reply:
Good catch, Dmitriy. Let me clarify this question. I think, this change is not blocking this PR. I guess I can do it separately after some clarification.